libp2p/network/url_conversion_test.go

```diff
@@ -0,0 +1,159 @@
+package network
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/multiformats/go-multiaddr"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPeerMultiAddrFromServiceURL_success(t *testing.T) {
```

Comment:
```suggestion
func TestPeerMultiAddrFromServiceURL_Success(t *testing.T) {
```

This is super helpful, ty!